specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/ServerAzureADAdministrators.json

```diff
@@ -234,49 +235,6 @@
           }
         }
       }
-    },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/disableAzureADOnlyAuthentication": {
```

Comment:
Hi @jamestao, Can I know why remove this? as it caused sdk breaking change, can we put deprecation info in description for some time then remove it in next new version?

Reply:
This API is not being used at all. functionality was never released publicly and there is no dependency. So, its better to remove it then have customers take any dependency on it.

Reply:
As per Amol, this was a limited, private preview api, and those customers were informed of the coming changes.